test/gui/shared/steps/steps.py

```diff
@@ -484,16 +485,23 @@ def step(context, publicLinkName, resource):
     expDate = []
     for row in context.table:
         if row[0] == 'expireDate':
-            expDate = row[1].split("-")
-    expYear = expDate[0][2:]
+            expDate = datetime.datetime.strptime(row[1],'%Y-%m-%d')
+    expYear = expDate.year-2000
```

Comment:
That's a bit sad that the app only uses 2-digit years. What happens if I want the public link to expire in 2101?

This code will have to be adjusted when we get near to 2100! :)